### PR TITLE
MKC: serene_sleuth.txt + support

### PR DESCRIPTION
**Pull request by Northmoc**

### CODE REVIEW

File: forge-game/src/main/java/forge/game/ability/effects/GoadEffect.java [hunk 1]
```diff
@@ -40,6 +41,11 @@ public void resolve(SpellAbility sa) {
                 continue;
             }
 
+            if (ungoad) {
+                tgtC.unGoad();
+                continue;
+            }
```
Comment (Hanmac) [inline forge-game/src/main/java/forge/game/ability/effects/GoadEffect.java:47]: @Northmoc what do you think? is this better?

Comment (Northmoc) [reply forge-game/src/main/java/forge/game/ability/effects/GoadEffect.java:47]: Yes – I think that's better! 😄

**Review by tool4ever — APPROVED** [review]

File: forge-game/src/main/java/forge/game/ability/effects/GoadEffect.java [hunk 2]
```diff
@@ -40,6 +41,11 @@ public void resolve(SpellAbility sa) {
                 continue;
             }
 
+            if (ungoad) {
+                tgtC.unGoad();
```
Comment (tool4ever) [inline forge-game/src/main/java/forge/game/ability/effects/GoadEffect.java:45]: I'm not sure if timestamps should be used here for those static ones like _Baeloth Barrityl_ 🤔

Comment (Northmoc) [reply forge-game/src/main/java/forge/game/ability/effects/GoadEffect.java:45]: Of course there are no release notes on this 💢 

My feel is something with power LT Baeloth's will be no longer goaded thanks to Serene Sleuth... until the state-based check makes it goaded again

Which I think would be correct with this code

Comment (Northmoc) [reply forge-game/src/main/java/forge/game/ability/effects/GoadEffect.java:45]: So I just tested this case –
Serene triggered for the Goblin, investigated, and removed the goad, but then Baeloth's static immediately re-goaded
I think this is good to ship for now until WotC clarifies, but I may be missing a rule
image

**Pull request merged**